crates/orderbook/openapi.yml

```
@@ -1038,6 +1054,33 @@ components:
             The UIDs of the orders included in the auction.
         prices:
           $ref: "#/components/schemas/AuctionPrices"
+    CompetitionOrderStatus:
+      type: object
+      properties:
+        type:
+          type: string
+          enum:
+            - Open
+            - Scheduled
+            - Active
+            - Solved
+            - Executing
+            - Traded
+            - Cancelled
+        value:
+          description: List of solvers and whether their solution includes the order.
+          type: array
+          items:
+            type: object
+            properties:
+              solver:
+                type: string
+                description: Name of the solver.
+              order_included:
```

To keep the same standard as other properties, please use camelCase:

```suggestion
              orderIncluded:
```